eventmesh-runtime/conf/eventmesh.yml

```diff
@@ -0,0 +1,141 @@
+      msgReqnumPerSecond: 15000
+      taskHandleExecutorPoolSize: 8
+    session:
+      # session expire time about client
+      expiredInMills: 60000
+      # flow control, include the global level and session level
+      upstreamBufferSize: 20
+    # for single event publish, maximum size allowed per event
+    maxEventSize: 1000
+    # for batch event publish, maximum number of events allowed in one batch
+    maxEventBatchSize: 10
+    global:
+      # thread number about global scheduler
+      scheduler: 5
+    # retry
+    retry:
+      async:
+        pushRetryTimes: 3
+        pushRetryDelayInMills: 500
+      sync:
+        pushRetryTimes: 3
+        pushRetryDelayInMills: 500
+      pushRetryQueueSize: 10000
+    # admin
+    admin:
+      http:
+        port: 10106
+    # registry
+    registry:
+      registerIntervalInMills: 10000
+      fetchRegistryAddrIntervalInMills: 20000
+#    # auto-ack
+#    defibus:
+#      client:
+#        consumeTimeoutInMin: 5
+    # sleep interval between closing client of different group in server graceful shutdown
+    gracefulShutdown:
+      sleepIntervalInMills: 1000
+    rebalanceRedirect:
+      sleepIntervalInMills: 200
+    # ip address blacklist
+    blacklist:
+      ipv4: 0.0.0.0/8,127.0.0.0/8,169.254.0.0/16,255.255.255.255/32
+      ipv6: ::/128,::1/128,ff00::/8
+    security:
+      enabled: false
+    trace:
+      enabled: false
+  sysid: 0000
+  # connector plugin
+  connector:
+    plugin:
+      type: standalone
+  # storage plugin
+  storage:
+    plugin:
+      type: standalone
+  # security plugin
+  security:
+    plugin:
+      type: security
+    validation:
+      type:
+        token: false
+    publickey: ''
+  # registry plugin
+  registry:
+    plugin:
+      enabled: false
+      type: nacos
+      server-addr: 127.0.0.1:8848
+      username: nacos
+      password: nacos
+  # metrics plugin, if you have multiple plugin, you can use ',' to split
+  metrics:
+    plugin: prometheus
+  # trace plugin
+  trace:
+    plugin: zipkin
+  # webhook
+  webHook:
+    admin:
+      # Start webhook admin service
+      start: true
+    # Webhook event configuration storage mode. Currently, only file and Nacos are supported
+    operationMode: file
+    fileMode:
+      # The file storage path of the file storage mode. If ${eventmeshhome} is written, it is in the eventmesh root directory
+      filePath: ${eventMeshHome}/webhook
+    # Nacos storage mode, and the configuration naming rule is eventmesh webHook. nacosMode. {nacos native configuration key} please see the specific configuration [nacos github api](https://github.com/alibaba/nacos/blob/develop/api/src/main/java/com/alibaba/nacos/api/SystemPropertyKeyConst.java)
+    ## Address of Nacos
+    nacosMode:
+      serverAddr: 127.0.0.1:8848
+    # Webhook eventcloud sending mode. And eventmesh connector. plugin. The type configuration is the same
+    producer:
+      storage: standalone
```

Comment:
![image](https://github.com/apache/eventmesh/assets/41445332/99683171-b442-47c2-92e3-24a9f81c9709)

As is seen in https://github.com/apache/eventmesh/issues/4185, According to the original Chinese comments in the git commit history, the existing English comments are misleading. The English translation does not match the meaning of the original Chinese text, which affects understanding.

```suggestion
    # Webhook CloudEvent sending mode. This property is the same as the eventMesh.storage.plugin.type configuration.
    producer:
      storage: standalone
```

Reply:
Thanks